src/components/ChatApp/MessageListItem/Feedback.react.js

```diff
@@ -95,7 +95,7 @@ class Feedback extends React.Component {
 				cursor: 'pointer'
 			}
 
-			let shareMessageSUSI = message.text;
+			let shareMessageSUSI = message.text === undefined ? "" : message.text;
```

Comment:
Please switch to single quotes so build passes. :sweat_smile:

Reply:
@anshumanv Made the changes :)